src/ErrorHandler.php

```diff
@@ -29,20 +29,20 @@ public function onError($code, $message, $file, $line)
         switch ($code) {
             case E_NOTICE:
             case E_USER_NOTICE:
-                $exc = new Errors\Notice($message, $file, $line, debug_backtrace());
+                $exc = new Errors\Notice($message, '', 0, debug_backtrace());
```

Comment:
Because `debug_backtrace()` already provides file, line and function. It is better to use it.